pwiz_tools/Skyline/TestUtil/Schemas/Skyline_Current.xsd

```diff
@@ -845,6 +855,22 @@
       </xs:extension>
     </xs:complexContent>
   </xs:complexType>
+  <xs:complexType name="protein_group_type">
+    <xs:complexContent>
+      <xs:extension base="doc_parent_node_type">
+        <xs:sequence>
+          <xs:element name="protein" type="protein_type" minOccurs="1" maxOccurs="unbounded" />
```

Comment:
Brendan mentioned in a call yesterday that Skyline was going to continue writing out a <protein> element when a group only has a single member. If that's correct, should this be minOccurs="2"?

Reply:
Yes, I think you're correct. Semantically, it should be 2.